src/qt/test/rpcnestedtests.cpp

```diff
@@ -13,8 +13,10 @@
 
 #include <llmq/context.h>
 
-#include <QDir>
-#include <QtGlobal>
+#include <QTest>
+
+#include <string>
+#include <stdexcept>
```

Comment:
21557: missing changes in `rpcNestedTest_rpc` probably due to missing backports

Reply:
one of the dependent backport is in https://github.com/dashpay/dash/pull/5936,  , will merge after this